File: PRNumberBiz/app/src/main/java/com/pplus/prnumberbiz/core/network/apis/INewApi.java
package com.pplus.prnumberbiz.core.network.apis;

import com.pplus.prnumberbiz.core.network.model.response.NewResultResponse;

import network.common.PplusCallback;
import retrofit2.Call;

/**
 * Created by j2n on 2016. 7. 25..
 * <p>
 * <pre>
 *     Application의 api interface를 정의합니다.
 * </pre>
 */
public interface INewApi{

    <T> void requestCall(Object tag, Call<NewResultResponse<T>> requestCall, PplusCallback<NewResultResponse<T>> responseCallback);
}

File: WowBox/app/src/main/java/com/lejel/wowbox/core/network/apis/INewApi.kt
package com.lejel.wowbox.core.network.apis

import com.lejel.wowbox.core.network.model.response.NewResultResponse
import com.pplus.networks.common.PplusCallback
import retrofit2.Call

/**
 * Created by j2n on 2016. 7. 25..
 *
 *
 * <pre>
 * Application의 api interface를 정의합니다.
</pre> *
 */
interface INewApi {
    fun <T> requestCall(tag: Any?,
                        requestCall: Call<NewResultResponse<T>>,
                        responseCallback: PplusCallback<NewResultResponse<T>>)
}